test/e2e/utils.go

```diff
@@ -0,0 +1,54 @@
+package e2e
```

Comment:
utils.go is an anti-pattern, please don't do it.

Reply:
i can rename it to helpers? We kinda already went this path all around the place.